### PR TITLE
S005-T79 - added GET method for transactions

### PR DESCRIPTION
**Pull request by Daractive**

### CODE REVIEW

**Review by Calharad — COMMENTED** [review]
Added some comments
Please check my small fix added directly to branch S005-T75 (pull request [here](https://github.com/AI-2020-2021-LAB1/stock-exchange-app-benchmark/pull/14) for quick info), I forgot to add something to filters. They match to query parameters from [there](http://193.33.111.196:8000/swagger-ui.html?urls.primaryName=default#/Stocks/getStocksUsingGET) (I hope you'll understand what do they mean, if not, write there or on Messenger)

File: benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/transaction/TransactionFiltersTO.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import java.util.Date;
+
+@Getter
+@Setter
+public class TransactionFiltersTO {
+    private String abbreviation;
+    private Long amount;
+    private Long amountMore;
+    private Long amountLess;
+    private Date date;
```
[the remark this answers is not in the document]
Comment (pabloducato) [reply benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/transaction/TransactionFiltersTO.java]: Resolved yesteday by @Daractive

File: benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/transaction/TransactionTO.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import com.project.benchmark.algorithm.dto.order.OrderTO;
+import java.util.Date;
+
+@Getter
+@Setter
+public class TransactionTO {
+    private long amount;
+    private OrderTO buyingOrder;
+    private Date date;
```
[the remark this answers is not in the document]
Comment (pabloducato) [reply benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/transaction/TransactionTO.java]: Resolved by @Daractive

**Review by pabloducato — APPROVED** [review]
Review ok

**Pull request merged**